### DISCUSSION TITLE
404 file not found - the Github-Pages do not show up

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#36332] [Other Feature Feedback, Questions, & Ideas]
hello dear all

the issue: 404 file not found - the Github-Pages do not show up

i have serious issues with the github - pages: i have no idea whats going on here - how to test and how to get more insights here?!

![image](https://user-images.githubusercontent.com/32641183/196050471-d9a9993d-357f-4ca6-8261-9f120fce6edc.png)

i need to have some options to test - and to find out what causes the issue!? The Pages just vanished

i have set the preliminary things in place: 

see
![image](https://user-images.githubusercontent.com/32641183/196053078-3a133286-b9d5-40f2-be55-0f0719beb8ed.png)

i am pretty sure that i need to....have a file named 'index.html' (all lower case) or '[README.md](https://readme.md/)' in the root level of the branch we selected as the publishing source for our repository.

**Comment by is-it-ayush** [comment]
Hey @fsj-digital! You need to have a `index.html` file for Github to pickup automatically and generate you a static page. 

Here is a link to the official refrence, where they explain you the process on how to set yourself a Github Page: [Here](https://pages.github.com/)

In Short:
You can get a Github Page by simply creating a repository with you username as the name of the repository. Then commit a index.html to it and the website (github page) will be available @ `<username>.github.io`
